control-plane/plugin/src/resources/pool.rs

```diff
@@ -60,18 +61,77 @@ impl GetHeaderRow for openapi::models::Pool {
     }
 }
 
+#[derive(Debug, Clone, clap::Args)]
+/// Arguments used when getting nodes.
+pub struct GetPoolsArgs {
+    /// Id of the node
+    #[clap(long)]
+    node_id: Option<NodeId>,
+
+    /// Filter pools on labels
```

Comment:
Would be nice if we could add support for all these, otherwise update accordingly.
```suggestion
    /// Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2).   
    /// Pools must satisfy all of the specified label constraints.
```